src/darwin/Framework/CHIP/MTRMetrics.h

```diff
@@ -32,16 +32,17 @@ MTR_NEWLY_AVAILABLE
 @property (nonatomic, nullable, readonly, copy) NSNumber * value;
 
 /**
- * Error code for the metric data. This value when valid, holds the error code value
- * of the operation associated with the event.
+ * Error code for the metric data. This value, when not nil, holds the error code value
```

Comment:
Is this the `MTRError` error code, or the internal SDK one?

Reply:
It can be any error. The actual interpretation is based on the metric key since it can be across may domains